index.html

```diff
@@ -10,6 +10,11 @@
 		<meta charset="utf-8" />
 		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
 		<link rel="stylesheet" href="assets/css/main.css" />

```

Comment:
You should no longer be using the main.css file if you have refactored everything to the other css files.